doc/api.yml

```diff
@@ -1181,6 +1181,115 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/students/{id}/pointing':
```

Comment:
For our use case, is this the appropriate use case ? Why it differs with the GET endpoint ?